app/src/main/res/values/styles.xml

```diff
@@ -1,7 +1,7 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
```

Comment:
You cannot really forced all the application to not have the Action Bar, was it the only solution ? Actually what was the problem with adding things to the existing action bar ?

Reply:
Corrected this ;)